src/coreComponents/mainInterface/ProblemManager.cpp

```diff
@@ -570,8 +575,11 @@ void ProblemManager::generateMesh()
         if( order > 1 )
         {
           MeshLevel & mesh = meshBody.createMeshLevel( MeshBody::groupStructKeys::baseDiscretizationString(),
-                                                       discretizationName,
-                                                       order );
+                                                       discretizationName, order );
+
+          CellBlockManager & highOrderCellBlockManager = meshBody.registerGroup< CellBlockManager >( keys::cellManager );
+
+          highOrderCellBlockManager.createHighOrderMaps( order, meshBody.getBaseDiscretization(), mesh );
```

Comment:
Due to visibility constraints, `createHighOrderMaps` won't be visible.

Reply:
Yes, createHighOrderMaps is removed from ProblemManger.cpp. However, with the current infrastructure, we do need to have an interface with CellBlockManger. Since the only visible class is the abstract class "CellBlockMangerABC", we have a virtual function "generateHighOrderMaps" defined at CellBlockMangerABC.hpp and calling from the MeshLevel constructor for high-order mesh-level. I hope this is satisfying as with the current infrastructure.